src/domain/file/create.js

```diff
@@ -22,11 +23,13 @@ async function create({
     tags,
     namespace
   }).save();
+
+  const storageLimit = getDefaultStorageLimit(contractAddress);
   await Limit.updateOne(
-    { contractAddress },
+    { invokerAddress, contractAddress },
```

Comment:
This is wrong. you need to take into account that there are portals where invoker address has not been set earlier, with this kind of setup it will create a new entry for these type of portals

Reply:
oh yes. That's true. 🤦🏻 
lemme look into that.